avocado/utils/nvme.py

```
@@ -529,3 +531,184 @@ def get_subsystem_using_ctrl_name(ctrl):
+        LOGGER.info("Executing command using pexpect: %s", cmd)
+        pexpect_handle = pexpect.spawn(cmd)
+        pexpect_handle.log_read = sys.stdout
+        for expect, value in list_of_expect_sendline:
+            pexpect_handle.expect(expect, timeout=30)
+            pexpect_handle.sendline(value)
+            LOGGER.debug("Matched String: %s",  pexpect_handle.after.strip())
+            LOGGER.debug("Pexpect output: %s", pexpect_handle.before.strip())
+            time.sleep(3)
+        pexpect_handle.close()
+        LOGGER.info("%s command executed successfully", cmd)
+    except pexpect.exceptions.TIMEOUT as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)